Add blog post about installer security release. (#63)
conda-forge/blog: posts/2023-07-13-installer-security-fixes.rst

.. post:: 2023-07-13
   :tags: security
   :author: conda-forge

.. role:: raw-html(raw)
   :format: html

Security updates to our installers
==================================

> In June 2023, software engineers from `Anaconda <https://www.anaconda.com>`__ have reported a security issue in the uninstallers that are included in the Windows versions of the `miniforge and mambaforge installers <https://github.com/conda-forge/miniforge>`__, one of the main ways to bootstrap conda-forge based conda & mamba distributions.

Reviewer: No strong opinion but ampersands don't look good in phrases IMO.

```suggestion
In June 2023, software engineers from `Anaconda <https://www.anaconda.com>`__ have reported a security issue in the uninstallers that are included in the Windows versions of the `miniforge and mambaforge installers <https://github.com/conda-forge/miniforge>`__, one of the main ways to bootstrap conda-forge based conda and mamba distributions.
```

Author: That's a valid opinion, I have no strong feelings

The issue could, under specific conditions, unintentionally delete files from your system during the uninstallation process. Anaconda has published more details in the related `blogpost <https://www.anaconda.com/blog/windows-installer-security-fix>`__ about the security fix for the miniconda and Anaconda Distribution Windows installers as well.

> Conda-forge is committed to fix the miniforge and mambaforge installers equally to reduce the possible impact on conda-forge users and has worked with Anaconda to mitigate the issue.

Reviewer 2: ```suggestion
conda-forge is committed to fix the miniforge and mambaforge installers equally to reduce the possible impact on conda-forge users and has worked with Anaconda to mitigate the issue.
```

> - As such, we are strongly recommending all users of miniforge and mambaforge to **update immediately** to the latest versions of miniforge and mambaforge. Please download them from the `miniforge repository's main page <https://github.com/conda-forge/miniforge>`__ or the `release specific page <https://github.com/conda-forge/miniforge/releases/tag/23.3.0-1>`__.

Reviewer 2: ```suggestion
- As such, we are strongly recommending all users of miniforge and mambaforge to **update immediately** to the latest versions of miniforge and mambaforge. Please download them from the `miniforge repository's main page <https://github.com/conda-forge/miniforge>`__ or the `release specific page <https://github.com/conda-forge/miniforge/releases/tag/23.1.0-4>`__.
```

> - For older versions, we are providing a **security patch for already installed miniforge and mamba installations**. You can download these from `release specific page <https://github.com/conda-forge/miniforge/releases/tag/23.3.0-1>`__ as well, under the names ``miniforge-uninstaller-patch-win-64-2023.07-0.exe`` and ``mambaforge-uninstaller-patch-win-64-2023.07-0.exe``.

Reviewer 2: I think there have been some issues with 23.3.0, because the last tag is 23.1.0 (even if the last files point to 23.3.0, the version is overridden by the tag, I think).

@isuruf @xhochy - can you clarify which Miniforge version is the one that is guaranteed to work upon a new release? 23.3.0 or 23.1.0?

Reviewer 3: `23.1.0`

> - For older versions, we are providing a **security patch for already installed miniforge and mamba installations**. You can download these from `release specific page <https://github.com/conda-forge/miniforge/releases/tag/23.3.0-1>`__ as well, under the names ``miniforge-uninstaller-patch-win-64-2023.07-0.exe`` and ``mambaforge-uninstaller-patch-win-64-2023.07-0.exe``.

Reviewer 2: ```suggestion
- For older versions, we are providing a **security patch for already installed miniforge and mambaforge installations**. You can download these from `release specific page <https://github.com/conda-forge/miniforge/releases/tag/23.1.0-4>`__ as well, under the names ``Miniforge3-uninstaller-patch-win-64-2023.07-0.exe`` and ``Mambaforge-uninstaller-patch-win-64-2023.07-0.exe``.
```

> - For older versions, we are providing a **security patch for already installed miniforge and mambaforge installations**. You can download these from `release specific page <https://github.com/conda-forge/miniforge/releases/tag/23.1.0-4>`__ as well, under the names ``Miniforge3-uninstaller-patch-win-64-2023.07-0.exe`` and ``Mambaforge-uninstaller-patch-win-64-2023.07-0.exe``.

Reviewer 2: ```suggestion
- For older versions, we are providing a **security patch for already installed miniforge and mambaforge installations**. You can download these from `release specific page <https://github.com/conda-forge/miniforge/releases/tag/23.1.0-4>`__ as well, under the names ``Miniforge3-uninstaller-patch-Windows-x86_64.exe`` and ``Mambaforge-uninstaller-patch-Windows-x86_64.exe``.
```

The CI is producing artifacts named like this ^

.. note::

>   To uninstall older versions of miniforge and mambaforge released before July XX, 2023, please download the security patch fix prior to uninstallation.

Reviewer 2: ```suggestion
  To uninstall older versions of miniforge and mambaforge released before July 1, 2023, please download the security patch fix prior to uninstallation.
```
